tests/wallet.rs

```diff
@@ -688,3 +688,20 @@ fn inscribe_gif() {
     ),
   )
 }
+
+#[test]
+fn wallet_balance() {
+  let rpc_server = test_bitcoincore_rpc::spawn_with(Network::Regtest, "ord");
+
+  CommandBuilder::new("--regtest wallet balance")
+    .rpc_server(&rpc_server)
+    .expected_stdout("0\n")
+    .run();
+
+  let _ = &rpc_server.mine_blocks(1);
```

Comment:
You can just do:

```suggestion
  rpc_server.mine_blocks(1);
```

`let _ =` is only needed when ignoring a type that has `#[must_use]` on it, like `Result`.

Reply:
right you are